util.py

```diff
@@ -27,6 +36,12 @@ def format_mac_address(mac_address):
     return re.sub('[:-]', '', mac_address).upper().strip()
 
 
+def datetime_to_timestamp_ms(dt):
```

Comment:
should we move datetime helpers to conduce-python-api from dhl-helpers so they can be used in dhl-pilot, dhl-ingest and conduce-python-api?

Reply:
Good catch.  I didn't know that was in both places.  I think it makes sense for the timestamp helpers to be in conduce-python-api.

We might just be able to remove them from dhl-helpers.